adaptive_hci/datasets.py

```diff
@@ -29,6 +30,33 @@
 ]
 
 
+def get_episode_modes(episodes, n_samples_considered: int = 20):
+    modes = []
+    for ep in episodes:
```

Comment:
when we have a leaderboard, check if mixing the episodes and taking the most common over that is better

Reply:
yes that should be the next step to do some sweep and find out